content-services/content-sources-backend
# Fixes 2676: add maximum retries for task

## Summary
This adds a "retries" column to the tasks table that counts the number of times a task has been requeued. If it has been requeued more than 3 times, the task fails.

## Testing steps
1. Create a repository with a large repo (like epel) that will take some time to sync instantly.
2. Exit the server with `Ctrl + C` so the sync task requeues
3. Do this 4 times. On the fourth time, there will be a warning that the task has failed.
4. Verify the results in the database.

## Checklist
